components/script/dom/readablestream.rs

```diff
@@ -562,10 +553,11 @@ impl ReadableStreamMethods for ReadableStream {
         }
         match self.reader {
             ReaderType::Default(ref reader) => {
-                reader.set(Some(&*ReadableStreamDefaultReader::new(
+                reader.set(Some(&*ReadableStreamDefaultReader::set_up(
                     &*self.global(),
                     self,
-                )));
+                    CanGc::note(),
```

Comment:
note: we should find a way to add the `can_gc` as an argument to `GetReader`.

Reply:
You can add something like 
```json
'ReadableStream': {
    'canGc': ['GetReader'],
},

```
in [`components/script/dom/bindings/codegen/Bindings.conf`](https://github.com/servo/servo/blob/main/components/script/dom/bindings/codegen/Bindings.conf)